### ISSUE TITLE
multi-polygon fountain doesn't render

### ISSUE THREAD

**Issue by fansanelli** [gravitystorm/openstreetmap-carto#4219]
Hello,

As described in subject, when a complex fountain is a multi-polygon relation won't render:

https://www.openstreetmap.org/relation/11734242#map=16/45.1376/7.6153

Can you please tell if can be fixed? Or is just better to avoid relation for these elements?

Many thanks
Best regards
Francesco

**Comment by pnorman** [comment]
It is rendering, see the label in the middle

![image](https://user-images.githubusercontent.com/1190866/95819824-b214a280-0cdb-11eb-839e-4414c229ec6f.png)

**Comment by fansanelli** [comment]
Yes! I meant to say that I would expect a different rendering... More like a building ☺️

**Comment by imagico** [comment]
The semantics of polygons tagged amenity=fountain vary - i documented that on the wiki now:

https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dfountain#Tagging

We therefore cannot render this with a styling implying anything more specific at this time.  Either rendering it as a water area or as a built structure of some sort would be wrong in a large number of cases.

Closing this as wontfix therefore - could be revisited if mapper decide on a specific meaning of the polygon.

What is definitely always correct and semantically clear at the moment is

* mapping fountains with a node at the main nozzle tagged amenity=fountain.
* mapping a standing water area that is part of a fountain as natural=water.
* micromapping elements of the fountain like barrier=wall.

Related to #1934.
